template/packages/localization/config/langs.ts

```diff
@@ -1 +1,18 @@
-export * from './langs.cjs';
+export const locales = {
+    EN: 'en-US',
+    CS: 'cs-CZ',
+} as const;
+
+/**
+ * @type {{ [key in keyof typeof locales]: import('type-fest').Split<(typeof locales)[key], '-'>[0]}
```

Comment:
should be only one brackets at the begining

Reply:
Nope, it’s correct but as I wrote above we can now extract it to normálně TS type.